adoc/helm_chart_override.adoc

```diff
@@ -12,14 +12,81 @@ You can adjust the {helm} chart installation in one of the following ways:
 [source]
 $ helm install <RELEASE_NAME> \
 oci://registry.suse.com/private-registry/private-registry-helm \
+--namespace <PRIVATE_REGISTRY_NAMESPACE> \
 --set harborAdminPassword=<MY_PASSWORD> \
---set externalURL=https://<MY_PRIVATE_REGISTRY_FQDN> \
---set expose.ingress.hosts.core=<MY_PRIVATE_REGISTRY_FQDN>
-* Create a custom `my-values.yaml` file and pass it to the `--values` flag, for example:
+--set externalURL=https://<PRIVATE_REGISTRY_FQDN> \
+--set expose.ingress.hosts.core=<PRIVATE_REGISTRY_FQDN>
+* Create a {suse} custom `suse_registry_override.yaml` file and pass it to the `--f` flag, for example:
 [source]
-$ helm install <RELEASE_NAME> \
-oci://registry.suse.com/private-registry/private-registry-helm \
---values my-values.yaml
+  $ helm install <RELEASE_NAME> \
+  oci://registry.suse.com/private-registry/private-registry-helm \
+  --namespace <PRIVATE_REGISTRY_NAMESPACE>
+  -f suse_registry_override.yaml
+
+[#pr-helm-chart-override-examples]
+=== Examples of {sregistry} {helm} override files
+.Minimal deployment with {ingress}
+[example]
+----
+expose:   
+  type: ingress //<1>
+  ingress:     
+    hosts:
+      core: <PRIVATE_REGISTRY_FQDN> //<2>
+
+externalURL: https://<PRIVATE_REGISTRY_FQDN> //<3>
+
+harborAdminPassword: "<MY_PASSWORD>" //<4>
+
+database:   
+  internal:     
+    password: "<MY_PASSWORD_POSTGRESQL>"
+
+redis:   
+  internal:     
+   password: "<MY_PASSWORD_REDIS>"
+----
+[callouts]
+<1> How {sregistry} is exposed. Can be `ingress`, `loadBalancer`, `nodePort` or `clusterIPhis`. Default is `ingress`.
+<2> Host name for the {kube} internal networking configuration.
+<3> URL where the {sregistry} application runs. It is used to generate links in the user interface, redirects and also for API responses.
+<4> The administrator password to the application.
+
+.Typical deployment with `loadBalancer`
+[example]
+----
+expose:   
+  type: loadBalancer //<1>
+  tls:
+    enabled: true
+    certSource: secret //<2>
+    secret:  
+      secretName: <SECRET_NAME>
+  
+    auto:
+      commonName: <PRIVATE_REGISTRY_FQDN> //<3>
+
+externalURL: https://<PRIVATE_REGISTRY_FQDN> //<4>
+
+harborAdminPassword: "<MY_PASSWORD>" //<5>
+
+database:   
+  internal:     
+    password: "<MY_PASSWORD_POSTGRESQL>"
+
+redis:   
+  internal:     
+   password: "<MY_PASSWORD_REDIS>"
+
+persistence: enabled //<6>
+----
+[callouts]
+<1> How {sregistry} is exposed. Can be `ingress`, `loadBalancer`, `nodePort` or `clusterIP`. Default is `ingress`.
+<2> Can be `auto`, `secret` or `none`. Depending on the option, you may have to include additional values.
+<3> To use a TLS encryption, update the field to match `externalURL`.
+<4> URL where the {sregistry} application runs. It is used to generate links in the user interface, redirects and also for API responses.
+<5> The administrator password to the application.
+<6>  To store all the images, metadata and scans, ensure this that the `persistence`-related settings are properly configured.
```

Comment:
> ensure this that the

Bad grammar here. I guess we meant "ensure that the".

Still, I feel like this comment confuses more than it helps. IIUC the persistence settings has already some sane defaults, so images, metadata and scans will be stored. The comment makes me believe that, unless I change those settings, things _won't_ be stored - which is not true?

I'd suggest to call out here _why_ a user would want to change the persistence settings. E.g., to increase the capacity? To use a different persistence backend? (I don't know the reasons, maybe @Mauraza you could provide some?